templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,237 @@
+          var event = new CustomEvent('mozUITour', {
+            bubbles: true,
+            detail: {
+              action: action,
+              data: data || {}
+            }
+          });
+
+          document.dispatchEvent(event);
+        }
+
+        function _generateCallbackID() {
+          return Math.random().toString(36).replace(/[^a-z]+/g, '');
+        }
+
+        function _waitForCallback(callback) {
+          var id = _generateCallbackID();
+
+          function listener(event) {
+            if (typeof event.detail != "object")
+              return;
+            if (event.detail.callbackID != id)
+              return;
+
+            document.removeEventListener("mozUITourResponse", listener);
+            callback(event.detail.data);
+          }
+          document.addEventListener("mozUITourResponse", listener);
+
+          return id;
+        }
+
+        Mozilla.UITour.DEFAULT_THEME_CYCLE_DELAY = 10 * 1000;
+        Mozilla.UITour.CONFIGNAME_SYNC = "sync";
+        Mozilla.UITour.CONFIGNAME_AVAILABLETARGETS = "availableTargets";
+
+        Mozilla.UITour.ping = function(callback) {
+          var data = {};
+          if (callback) {
+            data.callbackID = _waitForCallback(callback);
+          }
+          _sendEvent('ping', data);
+        };
+
+        Mozilla.UITour.observe = function(listener, callback) {
+          notificationListener = listener;
+
+          if (listener) {
+            document.addEventListener('mozUITourNotification',
+            _notificationListener);
+            Mozilla.UITour.ping(callback);
+          } else {
+            document.removeEventListener('mozUITourNotification',
+            _notificationListener);
+          }
+        };
+
+        Mozilla.UITour.registerPageID = function(pageID) {
+          _sendEvent('registerPageID', {
+            pageID: pageID
+          });
+        };
+
+        Mozilla.UITour.showHighlight = function(target, effect) {
+          _sendEvent('showHighlight', {
+            target: target,
+            effect: effect
+          });
+        };
+
+        Mozilla.UITour.hideHighlight = function() {
+          _sendEvent('hideHighlight');
+        };
+
+        Mozilla.UITour.showMenu = function(name, callback) {
+          var showCallbackID;
+          if (callback)
+            showCallbackID = _waitForCallback(callback);
+
+          _sendEvent('showMenu', {
+            name: name,
+            showCallbackID: showCallbackID,
+          });
+        };
+
+        Mozilla.UITour.hideMenu = function(name) {
+          _sendEvent('hideMenu', {
+            name: name
+          });
+        };
+
+        Mozilla.UITour.getConfiguration = function(configName, callback) {
+          _sendEvent('getConfiguration', {
+            callbackID: _waitForCallback(callback),
+            configuration: configName,
+          });
+        };
+
+        // custom
+        Mozilla.UITour.isTargetAvailable = function(target, track) {
```

Comment:
This is only ever called with `track=true`, should probably just remove that parameter.